src/Spreadsheet.tsx

```diff
@@ -133,7 +138,7 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
       if (value === undefined) {
         return "";
       }
-      return getValue({ ...point, data: value });
+      return getValue({ ...point, data: value as CellType });
```

Comment:
This `getValue` function seems to be a recurring pain point. It likely just needs to be typed to allow `ReactNode` which is a superset of everything.